tools/generate.py

```diff
@@ -339,9 +342,16 @@ def _build(parent: ObjectT, namespace: str, overrides: dict[str, str]) -> str:
     typings = "from typing import Any, Callable, Literal, Optional, Tuple, Type, TypeVar, Sequence"
 
     typevars: list[str] = []
-    imports: list[str] = []
+    imports: list[str] = [
```

Comment:
This will be included even when there are no deprecations but maybe that is fine.

Reply:
We include everything every time. A lot of typing imports are not necessary either, on most stubs. IMHO it is fine.